yarn.lock

```diff
@@ -4827,6 +4860,16 @@ lcid@^2.0.0:
   dependencies:
     invert-kv "^2.0.0"
 
+ledger-cosmos-js@^2.1.7:
```

Comment:
Same here, as we bundle ledger-cosmos-js for the time being, there shouldn't be mentions of ledger-cosmos-js in the lock file right ?